### PR TITLE
removed apostrophe

### PR DESCRIPTION
**Pull request by Soham-glitch**
Hey, I have created a pull request for issue no #1763 please review it

### CODE REVIEW

File: WSL/install.md [hunk 1]
```diff
@@ -89,11 +89,11 @@ WSL supports running as many different Linux distributions as you would like to
...
-4. From Windows Command Prompt or PowerShell, you can open your default Linux distribution inside your current command line, by entering: `wsl.exe`.
-5. From Windows Command Prompt or PowerShell, you can use your default Linux distribution inside your current command line, without entering a new one, by entering:`wsl [command]`. Replacing `[command]` with a WSL command, such as: `wsl -l -v` to list installed distributions or `wsl pwd` to see where the current directory path is mounted in wsl. From PowerShell, the command `get-date` will provide the date from the Windows file system and `wsl date` will provide the date from the Linux file system.
+- [Install Windows Terminal](/windows/terminal/get-started) ***(Recommended)*** Using Windows Terminal supports as many command lines as you would like to install and enables you to open them in multiple tabs or window panes and quickly switch between multiple Linux distributions or other command lines (PowerShell, Command Prompt, PowerShell, Azure CLI, etc). You can fully customize your terminal with unique color schemes, font styles, sizes, background images, and custom keyboard shortcuts. [Learn more.](/windows/terminal)
```
Comment (tuckstarsli) [inline WSL/install.md:92]: #1767

**Review by tuckstarsli — APPROVED** [review]

**Comment by Soham-glitch** [comment]
@microsoft-github-policy-service agree

**Comment by Soham-glitch** [comment]
I have agreed to the CLA. Firstly there was a little bit of trouble understanding that. 
Kindly review the changes.

**Review by mattwojo — APPROVED** [review]
Looks good. Thanks for contributing these updates @Soham-glitch

**Pull request merged**